### PR TITLE
RestaurantButton done

### PR DESCRIPTION
**Pull request by HadikaMalik**
I have completed the restaurant button and added props.

### CODE REVIEW

**Review by pedram-am — COMMENTED** [review]
Hi Hadika, well done! I've reviewed the changes you made in the Restaurant.jsx and RestaurantButton.jsx files, aligning with the task requirements to extract the button into a new component and pass the orderOne function as a prop. However, there seems to be a conflict in the Restaurant.jsx file due to changes made in the main branch. Please ensure you've pulled the latest updates from the main branch to resolve the conflict locally before updating your pull request. Once resolved, we can quickly merge the pull request. Great job! 👍

**Review by pedram-am — APPROVED** [review]
Hi Hadika! I've just made a little tweak in the Restaurant.jsx file to resolve the conflict. I removed the line "<RestaurantButton orderOne={orderOne} />". Also, I had a glance at the RestaurantButton.jsx file, and it looks fantastic! Your work on it is really impressive. 👏

**Comment by HadikaMalik** [comment]
Thank you Pedram for your kind words.

File: src/components/Restaurant/Restaurant.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import "./Restaurant.scss";
+import RestaurantButton from "../RestaurantButton/RestaurantButton";
```
Comment (eliza-an) [inline src/components/Restaurant/Restaurant.jsx:3]: You import this but this button is never used I believe?

**Review by eliza-an — COMMENTED** [review]
Perhaps i misunderstood the task, but I believe you should have made a new component called restaurantbutton, and used that component instead of the current button component on line 17. Otherwise, you created a restaurant button component and did not use it!

Also, I did not see a test for restaurant button :)
image

**Pull request merged**